pkgs/by-name/wo/wordlists/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, nmap
+, rockyou
+, runtimeShell
+, seclists
+, symlinkJoin
+, tree
+, wfuzz
+, lists ? [
+    nmap
+    rockyou
+    seclists
+    wfuzz
+  ]
+}:
+
+symlinkJoin rec {
+  pname = "wordlists";
+  version = "unstable-2023-10-10";
+
+  name = "${pname}-${version}";
+  paths = lists;
+
+  postBuild = ''
+    # Create a command to show the location of the links.
+    mkdir -p $out/bin
+    cat >> $out/bin/wordlists << __EOF__
+    #!${runtimeShell}
+    ${tree}/bin/tree $out/share/wordlists
+    __EOF__
+    chmod +x $out/bin/wordlists
+
+    # Create a handy command for easy access to the wordlists.
+    # e.g.: `cat "$(wordlists_path)/rockyou.txt"`, or `ls "$(wordlists_path)/dirbuster"`
+    cat >> $out/bin/wordlists_path << __EOF__
+    #!${runtimeShell}
+    echo $out/share/wordlists
```

Comment:
What happens if there is no coreutils in path and bash is used. `echo` is not a built-in in bash. You could use `printf` instead.

Reply:
When using printf the string has a `%` at the end and I don't know why. Also I would assume that every environment has some form of coreutils, for example if you use nixos they are installed no matter what since we use them in the initrd and most shells like zsh have echo builtin anywyas, so I would just leave it as is if it's not a blocker for you.

Reply:
When using printf the string has a `%` at the end and I don't know why. Also I would assume that every environment has some form of coreutils, for example if you use nixos they are installed no matter what since we use them in the initrd and most shells like zsh have echo builtin anywyas, so I would just leave it as is if it's not a blocker for you.